### PR TITLE
added layout for end of quiz

### PR DESCRIPTION
**Pull request by jvresk**

### CODE REVIEW

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            android:textColor="@android:color/black"
+            android:textSize="22dp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/congratulation" />
+
+        <ImageView
+            android:id="@+id/fireworks"
+            android:layout_width="180dp"
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:41]: 175 or 200 please :)

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+        <TextView
+            android:id="@+id/achieved_score"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="X "
+            android:textColor="@color/colorPrimary"
+            android:textSize="22dp"
+            android:textStyle="bold"
+            android:layout_marginTop="10dp"
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:71]: 25dp

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+        <TextView
+            android:id="@+id/achieved_titles"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="X "
+            android:textColor="@color/colorPrimary"
+            android:textSize="22dp"
+            android:textStyle="bold"
+            android:layout_marginTop="10dp"
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:99]: 25dp

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/titles_text" />
+
+        <Button
+            android:id="@+id/end_quiz"
+            android:layout_width="match_parent"
+            android:layout_height="40dp"
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:107]: 50dp or wrap content should be better

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+            android:text="X "
+            android:textColor="@color/colorPrimary"
+            android:textSize="22dp"
+            android:textStyle="bold"
+            android:layout_marginTop="10dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/titles_text" />
+
+        <Button
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:104]: This should be replaced with a material design button, refer to the login / register screen

File: CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+            android:background="@drawable/round_corners"
+            android:gravity="center"
+            android:text="Continue"
+            android:textSize="18dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/achieved_titles" />
+
+        <View
```
Comment (vhrvoje2) [inline CircuitMessing/app/src/main/res/layout/end_quiz_fragment.xml:117]: What is the purpose of this view?

**Review by vhrvoje2 — CHANGES_REQUESTED** [review]
Please implement the requested changes or leave feedback to my comments

**Comment by bzitkovic** [comment]
> Please implement the requested changes or leave feedback to my comments

Yes, the changes that Hrvoje suggests would be nice.

**Comment by jvresk** [comment]
I will change that.
View is the horizontal divider, mistake. Removing it.

**Pull request merged**